app/controllers/concerns/can_redirect_form_after_renew.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module CanRedirectFormAfterRenew
+  extend ActiveSupport::Concern
+
+  included do
+    private
+
+    def redirect_after_successful_submit
+      if renew_if_possible
```

Comment:
Just wondering about whether this needs to be in its own concern?

I'm assuming we will only ever use this in conjunction with `CanRenewIfPossible`, and that in most cases where `CanRenewIfPossible` will be used we'll want to redirect afterwards anyway.

So is it worth just adding `redirect_after_successful_submit()` to `CanRenewIfPossible`?